### PR TITLE
DOCS: Also exclude docs/ branches from CI pipelines

### PR DESCRIPTION
**Pull request by tkfu**
@lbonn: I promise this is the last one :-P. (I will likely backport it, but I'll just push directly rather than spamming with PRs.)

### CODE REVIEW

File: ci/gitlab/.gitlab-ci.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ Docker Setup:
   stage: docker
   except:
     - /^20\d\d\.\d\d?-docs$/
+    - /^docs/
```
Comment (lbonn) [inline ci/gitlab/.gitlab-ci.yml]: Should we use `/^docs\//`?

Comment (tkfu) [reply ci/gitlab/.gitlab-ci.yml]: Good call, fixed.

**Comment by patrickvacek** [comment]
I'm confused, is this preventing the docs getting added from any of the docker images? We just recently explicitly added building the Doxygen to one of our pipelines to make sure that doesn't get accidentally broken. It looks like this broke that check if I'm not mistaken.

**Comment by patrickvacek** [comment]
> is this preventing the docs getting added from any of the docker images?

Ah, nm, sorry, this is filtering based on branch name. My bad.

**Pull request merged**